src/GraphQL/Language/CoreToVanillaConverter.cs

```diff
@@ -13,18 +12,12 @@ namespace GraphQL.Language
 {
     public class CoreToVanillaConverter
     {
-        private readonly ISource _body;
```

Comment:
I was surprised when I saw that this field is not being used. The class is stateless and can generally be made static. I declared a static instance. Maybe make the class static?

Reply:
done